src/frontend/js/modules/header/header.spec.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+/* global chai: false */
+/* global sinon: false */
+
+var { expect } = chai;
+
+describe('The mainHeader directive', function() {
+  var $rootScope, $compile;
+  var $scope;
+
+  beforeEach(function() {
+    angular.mock.module('esn.header');
+  });
+
+  beforeEach(angular.mock.inject(function(
+    _$rootScope_,
+    _$compile_,
+  ) {
+    $rootScope = _$rootScope_;
+    $compile = _$compile_;
+  }));
+
+  function initDirective(html, scope) {
+    var element = $compile(html || '<main-header />')(scope);
+    scope.$digest();
+
+    return element;
+  }
+
+  it('should show global search box if showGlobalSearch is true', function() {
+    $scope = $rootScope.$new();
+    var element = initDirective(
+      '<main-header />',
+      $scope
+    );
+
+    $scope.showGlobalSearch = true;
+    $scope.$digest();
+
+    expect(element.find('.search-header').length).to.not.equal(0);
+  });
+
+  it('should NOT show global search box if showGlobalSearch is false', function() {
+    $scope = $rootScope.$new();
+    var element = initDirective(
+      '<main-header />',
+      $scope
+    );
+
+    $scope.showGlobalSearch = false;
+    $scope.$digest();
+
+    console.log('#################', element);
```

Comment:
```suggestion
```

Reply:
ah, my bad. Tried to commit before leaving the building 🤣

Reply:
I will fix it tomorrow